Week3/more_javascript.js

```diff
@@ -0,0 +1,183 @@
+/*1- Create a function that takes 3 arguments and returns the sum of the these arguments.*/
+function sum(x, y ,c) {
+   return x+y+c;
+}
+console.log(sum(2,3,5)); 
+
+/* 2- Create a function named colorCar that receives a color, and prints out, 'a red car' for example.*/
+function colorCar(color){
+console.log("a %o car", color)
+}
+colorCar("red");
+
+/* 3- Create an object and a function that takes the object as a parameter and prints out all of its properties and values.*/
+
+let myDetails = {
+   Name : "Mosleh",
+   Age : 30,
+   Hoppies: ["bascketball" , "movies"]
+}
+function aboutMe(myDetails){
+return console.log(myDetails);
+}
+aboutMe(myDetails);
+// I tried also to get this result using for loop 
+// for (let key in myDetails){
+//    console.log(key + ": " +myDetails[key]);
+// }
+
+/* 4-Create a function named vehicleType that receives a color, and a code, 1 for car, 2 for motorbike. And prints 'a blue motorbike' for example when called as vehicleType("blue", 2)*/
+
+function vehicleType(color, code){
+  if (code === 1){
+     console.log("a %o car", color )
+   
+   }else if (code ===2)
+   console.log("a %o motorbike", color)
+}
+   vehicleType("blue",1);
+   vehicleType("red",2);
+
+/* 5- Can you write the following without the if statement, but with just as a single line with console.log(...);?
+if (3 === 3) {
+  console.log("yes");
+} else {
+  console.log("no");
+}*/
+
+switch (3===3){
+   case (true):
+      console.log('Yes');
+      break;
+default:
+      console.log('No');
+}
+
+/* 6- Create a function called vehicle, like before, but takes another parameter called age, so that vehicle("blue", 1, 5) prints 'a blue used car'*/
+
+function vehicle(color, code, age){
+   if (code === 1 && age > 0 ){
+      console.log("a %o used car", color)
+  } else if (code ===2 && age > 0)
+    console.log("a %o used motorbike", color)
+    else if (code ===1 && age === 0)
+    console.log("a %o new motorbike", color)
+    else if (code ===2 && age === 0)
+    console.log("a %o new motorbike", color)
+}
```

Comment:
Here the same comment as before (with the `identation`). And can you also use `{` and `}` around your `else if` statements. Because it's easy to make little mistakes in this way.

And I think that this assignment is supposed to use the `myVehicles` array. So can you take a look at this again?

Reply:
I think that we have to use `myVehicles` array in step 7, is it right?